lib/models.ts

```diff
@@ -26,12 +26,7 @@ export interface ParsedComponentsDefinition {
                     tag: string;    // tag name (lowercased)
                 };
             };
-            properties: {
-                [name: string]: {
-                    property: ComponentsDefinition['componentProperties'][0];
-                    directiveKey: string | null;
-                };
-            };
+            properties: ComponentsDefinition['componentProperties'][0][];
```

Comment:
Should we add a type for `ComponentsDefinition['componentProperties'][0]`?

Reply:
This one is the autogenerated type from the schema. You can grab the nested type like this.

Reply:
It's just reads a bit awkward